instruments/tests/test_keithley/test_keithley580.py

```diff
@@ -761,7 +761,7 @@ def test_measure(init, create_measurement, resistance):
             sep="\n"
     ) as inst:
         read_value = inst.measure()
-        assert read_value.magnitude == pytest.approx(resistance, rel=1e-6)
+        assert read_value.magnitude == pytest.approx(resistance, rel=1e-5)
```

Comment:
Did this one fail again when you tested? Really strange... 
Turning the comparison around would have solved it last time too (your proposed that) but I don't understand why... just the way comparisons are done?

Reply:
Naw turning it around in this case actually wouldn't work, I was misremembering something. It works when you're comparing unitful values where the left and right side might have different SI prefixes. Being mindful of unseen division operations that are happening can be important because they're the ones that usually end up showing floating point errors.

Now for this in particular, `rel=1e-6` on a resistance value of `1e-3` was (correctly) giving a resulting absolute bound of `1e-9` which is slightly too small for this rounding error.

Reply:
Oh, good point. That means it reports out absolute errors in the error message, so `pytest` worked fine and you're right to make it 1e-5 to give it a bit more. Thanks for clarifying, this makes a lot more sense now :smile: